### DISCUSSION TITLE
Increase the verbosity of files fixed from a formatting run

### DISCUSSION THREAD

**Discussion by martinjlowm** [biomejs/biome#2276] [Ideas]
I was troubleshooting some formatting errors with a colleague today and we didn't find the reporting of "Fixed N files" very helpful from a CI job. (the root cause was really a stale pre-commit configuration).

Surely, if you are fixing thousands of files, that won't be super helpful either, so a sane upper limit similar to the reported lint diagnostics would be neat.

```
Fixed X files, of which Y were:
- A
- B
- C
```

**Comment by arendjr** [comment]
Would a summary as discussed here be of help? https://github.com/biomejs/biome/issues/57

**Reply by martinjlowm** [reply]
I believe it would. As long as that summary would include a sane number of files. Perhaps with a programmatic output (thinking JSON), the full list could be presented.

**Reply by ematipico** [reply]
They seem to be two feature requests:
- present a summary for each command 
- dump the output in a format different from just words

They are both tracked.
